### PR TITLE
USHIFT-377: make sure all files from assets/ are handled by rebase

### PR DESCRIPTION
**Pull request by pmtk**

### CODE REVIEW

**Comment by pmtk** [comment]
Builds on top of https://github.com/openshift/microshift/pull/1347

File: scripts/auto-rebase/presubmit.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        sys.exit(1)
+
+    recipe_filepath = "./scripts/auto-rebase/assets.yaml"
+    recipe = yaml.load(open(recipe_filepath).read(), Loader=Loader)
+    assets_filelist = build_assets_filelist_from_recipe(recipe)
+
+    realfiles =  [ f.removeprefix('assets/') for f in glob.glob('assets/**/*.*', recursive=True) ]
+    missing = []
+    for f in realfiles:
+        if f not in assets_filelist:
```
Comment (dhellmann) [inline scripts/auto-rebase/presubmit.py:60]: Should we check for the reverse condition (where the file is in the asset list but not the repo), too?

Comment (pmtk) [reply scripts/auto-rebase/presubmit.py]: Great idea, done!

**Pull request merged**